File: tourneyAPI/Models/Entities/Characters/Mewtwo.cs


namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Mewtwo : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Mewtwo()
    {
        Id = CharacterId.Mewtwo;
        characterName = CharacterName.MEWTWO;
        archetype = Archetype.GLASS_CANNON;
        fallSpeed = FallSpeed.FLOATY;
        weightClass = WeightClass.FEATHERWEIGHT;
        tierPlacement = TierPlacement.C;
    }

}

File: tourneyClient/src/models/entities/Characters/Mewtwo.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Mewtwo: Character =
{
    id: CharacterId.Mewtwo,
    characterName: CharacterName.MEWTWO,
    archetype: Archetype.GLASS_CANNON,
    fallSpeed: FallSpeed.FLOATY,
    weightClass: WeightClass.FEATHERWEIGHT,
    tierPlacement: TierPlacement.C
}

export default Mewtwo; 